docs/concepts/cluster_queue.md

```diff
@@ -69,7 +69,7 @@ An example of a ClusterQueue with codependent resources looks like the following
 apiVersion: kueue.x-k8s.io/v1alpha2
 kind: ClusterQueue
 metadata:
-  name: cluster-total
+  name: cluster-queue
```

Comment:
I think we can keep this one. The name attempts to reflect that there is one just one cluster queue in this cluster. Otherwise, change the name in line 17, to keep names consistent.

Reply:
I'll change the name in line 17... I prefer the more descriptive name.